website/views/shared/modals/quests.jade

```diff
@@ -90,3 +77,17 @@ script(type='text/ng-template', id='modals/questInvitation.html')
     button.btn.btn-default(ng-click='questHold = true; $close()')=env.t('askLater')
     button.btn.btn-default(ng-click='questReject(party); $close()')=env.t('reject')
     button.btn.btn-primary(ng-click='questAccept(party); $close()')=env.t('accept')
+
+script(type='text/ng-template', id='modals/questDrop.html')
+  .quest-icon.pull-right(class='inventory_quest_scroll_{{::selectedQuest.key}}')
+  .modal-header
+    h4=env.t('messageDropQuest')
+  .modal-body
+    h4 {{::selectedQuest.text()}}
+    +questInfo
+    hr
+    .npc_ian.pull-left
+    p=env.t('dropQuestCongrats')
+  .modal-footer
+    button.btn.btn-default(ng-click='closeQuest(); $close()')=env.t('questLater')
+    button.btn.btn-primary(ng-click='questInit(); $close()')=env.t('inviteParty')
```

Comment:
I don't think you're passing in a controller when opening this modal. Would it be helpful to pass in the partyCtrl and create a function on `$scope` there that does the `inviteOrStartParty` code? This is how it's done in the Header controller: https://github.com/HabitRPG/habitrpg/blob/32a74471dbeb791fe5fafc6d5441e5cda827968b/website/public/js/controllers/headerCtrl.js#L9